compatibility-tests/v1_31/src/lib.rs

```diff
@@ -1 +1,97 @@
+#![cfg(test)]
+
 // There are not yet any feature differences that require testing for Rust 1.31 compatibility.
```

Comment:
Out of curiosity, what are these new v1.31 "*_style" modules checking for?  It seems like a basic functionality test, but I'm not sure how it relates to 1.31 specifically.

Is this comment still accurate about having no feature differences to check?

Reply:
Yeah, still no feature differences. 

I realized that this test was checking that `snafu` and `snafu-derive` would compile in Rust 1.31, but _wasn't_ testing that the code **generated** by the macro would compile in Rust 1.31 because it wasn't generating any code. This just adds some basics.